### ISSUE TITLE
[Bug]: DetailsList Accessibility Insights FastPass failures - CellSizer & Expand/Collapse

### ISSUE THREAD

**Issue by gfisher-vmsft** [microsoft/fluentui#26924]
### Library

React / v8 (@fluentui/react)

### System Info

```shell
All browsers
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist/grouped

## Repro Steps
1. Navigate to [v8 DetailsList - Grouped example](https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist/grouped)
2. Run FastPass' automated checks

## Actual Behavior
- `aria-required-children` error from cellSizer with role="button" nested within parent with role="row". 
- `aria-required-children` error from an invisible element with `data-icon-name="Tag"`
- `aria-allowed-attr` error from div with role="row" and `aria-expanded="true"` nested inside parent with `role="grid"`

## Expected Behavior
No errors picked up by Accessibility Insights' FastPass automated checks

## Notes
- Is the `Tag` issue just leftover code? In `GroupHeader.base.tsx`, there is a div with className of `dropIcon` that wraps `<Icon iconName="Tag">`. The CSS makes this invisible. I'm not sure what value this element is adding. 

### Requested priority

Blocking

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by axlvlv** [comment]
I'm facing the same issue for this accessibility error

**Comment by ClementCHA** [comment]
I'm currently facing the same issue with my detailList. 

Have this issue since Accessibility Insights upgrade to **2.37.3**

**Comment by TheWebartist** [comment]
I'm also facing the same issue for this accessibility error. For all basic List controls :
`aria-required-children` error from cellSizer.
Thanks for your work

**Comment by rogerprz** [comment]
Hello any update?

**Comment by smhigley** [comment]
Hi! The `aria-required-children` error is a false positive that has already been fixed by axe-core: https://github.com/dequelabs/axe-core/issues/3850

The `aria-allowed-attr` error is technically disallowed from `role=row` when inside a `role=grid` element, but we have it on there because of a combination of circumstances:
1. The Grouped DetailsList funcitons more like a grid than a treegrid, so unlike GroupedList, we decided to keep grid semantics to improve virtual cursor table navigation support.
2. For historical reasons, v8 DetailsList places focus on rows and not on cells. This means semantic information needs to be exposed on rows to support forms mode/app mode screen reader navigation.
3. Putting `aria-expanded` on a row does no harm, and does not negatively impact any user experience.

Both these bugs should be marked as false positives if found through testing, or bypass those tests on those selectors if using axe-core in a testing suite.
